cli/src/main.rs

```diff
@@ -166,6 +166,11 @@ fn process_apply(
         );
         return Ok(());
     }
+    if confirm {
+        println!(
+        "\nThank you for creating your on-chain registration for the Solana Foundation Delegation Program.\nAs a reminder, your registration is not complete until you signup on https://solana.foundation/delegation-program"
+        );
+    }
```

Comment:
I think this belongs at the end of the body of the `process_apply` function, so that it is only shown after the registration is successful. The way it is now, you could get the "thank you" message even if registration failed for some reason.

Also, you don't need the `if confirm` conditional.

Reply:
should it be at the bottom of `process_apply` or `send_and_confirm_message`?

Reply:
It should be at the end of `process_and_apply`; `send_and_confirm_message` is a generic message used by other functions to do things other than apply.

To do this you'll have to create a variable to hold the value returned by `send_and_confirm_message`, and then return that value after printing the message.

Or, alternately, you could add the print statement after `process_apply` is called, here: https://github.com/solana-labs/stake-o-matic/pull/253/files#diff-37970e7d817d4316a093bb967d74ff7e26d9c6c01d93f5a79d3d195ba9f6155bL704. That might be easier.

Either way, let me know if you have any questions.